sdcm/utils/k8s.py

```diff
@@ -166,6 +173,34 @@ def deploy(self):
         self.service_ip = service_ip
         self.is_deployed = True
 
+    def is_deleted(self):
+        try:
+            self.service
+        except RuntimeError:
+            return True
+        return False
+
+    def delete(self):
+        n, ns = self.name, self.namespace
+
+        try:
+            LOGGER.debug(f"Trying to delete '{n}' K8S service in the '{ns}' namespace")
+            self.core_v1_api.delete_namespaced_service(
+                name=self.name, namespace=self.namespace, async_req=False)
```

Comment:
did you assign n, ns variable before try..except clause to use them here?

Reply:
Done